apps/jetpackme-support/index.js

```diff
@@ -0,0 +1,29 @@
+import OdieAssistantProvider, {
```

Comment:
Nit: I'd prefer if we moved this into an `src/` directory, just for consistency between this and other apps and packages in the repo. Also handy in case we eventually need to add more code here.